inference/server/oasst_inference_server/user_chat_repository.py

```diff
@@ -17,7 +17,7 @@ class Config:
     async def get_chats(self) -> list[models.DbChat]:
         query = sqlmodel.select(models.DbChat)
         query = query.where(models.DbChat.user_id == self.user_id)
-        query = query.order_by(models.DbChat.created_at.desc())
+        query = query.order_by(models.DbChat.id.desc())
```

Comment:
could we leave this sorted by date? this has the advantage of being functional regardless of which id type we use. Also perhaps slightly faster but not that much.

Reply:
Done!